core/src/test/java/org/apache/iceberg/TableTestBase.java

```diff
@@ -237,12 +253,23 @@ ManifestFile writeManifest(DataFile... files) throws IOException {
   }
 
   ManifestFile writeManifest(Long snapshotId, DataFile... files) throws IOException {
-    File manifestFile = temp.newFile("input.m0.avro");
```

Comment:
do we need to maintain/keep that `input.m0.avro` file name?

Reply:
I'm pretty sure this is needed, because we're determining the file format from the file ending

Reply:
Yes, we do need the manifest file to end in ".avro", but it's fine to call `temp.newFile()` (which uses a random name) and add the extension afterwards.